pyproject.toml

```diff
@@ -74,6 +74,7 @@ filterwarnings = [
   "ignore:the imp module is deprecated in favour of importlib:DeprecationWarning",  # move to importlib to fix (setuptools issue)
   "ignore:The NumPy module was reloaded:UserWarning",
   "ignore:.*np\\.MachAr.*:DeprecationWarning",  # until https://github.com/numba/numba/issues/7758 is fixed
+  "ignore:module 'sre_.*' is deprecated:DeprecationWarning", # From lark & Python 3.11
```

Comment:
FYI, this can also be "default", "once", or "all" if you want the warning to still show up in the tests. But they aren't really our fault so I think ignoring them is fine. But there are options.